libr/include/r_util/r_annotated_code.h

```diff
@@ -21,44 +21,43 @@ typedef enum r_syntax_highlight_type_t {
 	R_SYNTAX_HIGHLIGHT_TYPE_GLOBAL_VARIABLE,
 } RSyntaxHighlightType;
 
-/**
- * enum r_code_annotation_type_t - typedefed as RCodeAnnotationType and this gives types of annotation
- *
- * There are two kinds of RCodeAnnotation. One for offset, which of the type 
- * R_CODE_ANNOTATION_TYPE_OFFSET and other one is for syntax highlight, which is
- * of the type R_CODE_ANNOTATION_TYPE_SYNTAX_HIGHLIGHT.
- * R_CODE_ANNOTATION_TYPE_OFFSET is for representing annotations that gives an offset for 
- * a range while R_CODE_ANNOTATION_TYPE_SYNTAX_HIGHLIGHT is for representing the
- * kind of data the range represents. Here, range refers to the range of annotation.
- */
-
+//! Represents the type of annnotation.
```

Comment:
In most cases Doxygen treats `//!` `///` `/**` and `/*!` almost identically so it's a question of chosen style. Whatever style is chosen use it consistently. Everywhere else in this file you used `/**` version so you should use the same for this enum instead of `//!` or `/*!`.

Reply:
Done.